### DISCUSSION TITLE
Glade crashing on FC35 when using Anaconda windows

### DISCUSSION THREAD

**Discussion by jdoxey** [rhinstaller/anaconda#3707] [Q&A] [answered]
Hi community,
I'm writing an Anaconda add-on which includes GUI screens and Glade is crashing when adding Anaconda widgets. I'm on Fedora 35, and I have installed the package `anaconda-widgets-devel`. I'm using the default version of Glade for Fedora 35 which is currently 3.38.2.
Glade itself is opening fine, and adding an AnacondaDiskOverview widget to a regular GtkWindow works fine. But Glade crashes when I try to add an AnacondaSpokeWindow.
I have submitted a crash report (which goes to https://retrace.fedoraproject.org/faf/reports/90252/), just wondering if people know of any workarounds?

**Comment by VladimirSlavik** [comment, marked as answer]
I don't have any hard data, but on my own laptop with Fedora 32, it works well enough. So perhaps a VM for the changes you want?

Just a hunch: Try loading under a Xorg session instead of Wayland, if you have the option. There are issues with the keyboard switcher, which appears to be part of the window, and under X it works, so maybe that could be it?

**Reply by jdoxey** [reply]
Yes! It doesn't crash if I choose "GNOME on Xorg" when logging in. Good hunch. Thank you.
